custodial-copy-backend/src/test/scala/uk/gov/nationalarchives/custodialcopy/testUtils/ExternalServicesTestUtils.scala

```diff
@@ -678,6 +724,8 @@ object ExternalServicesTestUtils extends MockitoSugar with EitherValues {
       repTypeCaptor.getAllValues.asScala.toList should equal(repTypes)
       repIndexCaptor.getAllValues.asScala.toList should equal(repIndexes)
 
+      if numOfStreamBitstreamContentCalls > 0 then fileObjectUrl.getAllValues.asScala.toList should equal(List.fill(numOfStreamBitstreamContentCalls)("url"))
```

Comment:
Is this effectively just checking the size of the lists, if all elements in the expected list are hardcoded to `url` ?

Reply:
I've removed the `if numOfStreamBitstreamContentCalls > 0 then ` as it's not really necessary